### ISSUE TITLE
Segmentation fault in the online_demo code

### ISSUE THREAD

**Issue by rachillesf** [mit-han-lab/temporal-shift-module#7]
Hi, thanks for sharing this work

I'm having a segmentation fault when running online_demo code. Here is the error:
~~~~
UserWarning: The use of the transforms.Scale transform is deprecated, please use transforms.Resize instead.
  warnings.warn("The use of the transforms.Scale transform is deprecated, " +
Build Executor...
Segmentation fault (core dumped)
~~~~

I found that the program is crashing in line 35 of the main.py file and I'm currently using the LLVM-4.0.0 on ubuntu 16.04.
~~~~
relay_module, params = tvm.relay.frontend.from_onnx(onnx_model, shape=input_shapes)
~~~~

Can anyone replicate this problem?

Thanks for your help.

**Comment by idy002** [comment]
Hi,
This line transforms the onnx model to tvm relay module.
1. Can you check that your tvm and onnx have been installed correctly and are the latest version?
2. Also, you can debug the program and step into this function to figure out which part causes the problem.

**Comment by karenz17** [comment]
I have the same problem,

> Hi, thanks for sharing this work
> 
> I'm having a segmentation fault when running online_demo code. Here is the error:
> 
> ```
> UserWarning: The use of the transforms.Scale transform is deprecated, please use transforms.Resize instead.
>   warnings.warn("The use of the transforms.Scale transform is deprecated, " +
> Build Executor...
> Segmentation fault (core dumped)
> ```
> 
> I found that the program is crashing in line 35 of the main.py file and I'm currently using the LLVM-4.0.0 on ubuntu 16.04.
> 
> ```
> relay_module, params = tvm.relay.frontend.from_onnx(onnx_model, shape=input_shapes)
> ```
> 
> Can anyone replicate this problem?
> 
> Thanks for your help.

I have the same problem, how do you solve it?

**Comment by tonylins** [comment]
Hi, it is hard to diagnose the problem with just segmentation fault. Could you please provide more error log regarding the problem, or provide a minimal script to reproduce the error?

**Comment by karenz17** [comment]
> Hi, thanks for sharing this work
> 
> I'm having a segmentation fault when running online_demo code. Here is the error:
> 
> ```
> UserWarning: The use of the transforms.Scale transform is deprecated, please use transforms.Resize instead.
>   warnings.warn("The use of the transforms.Scale transform is deprecated, " +
> Build Executor...
> Segmentation fault (core dumped)
> ```
> 
> I found that the program is crashing in line 35 of the main.py file and I'm currently using the LLVM-4.0.0 on ubuntu 16.04.
> 
> ```
> relay_module, params = tvm.relay.frontend.from_onnx(onnx_model, shape=input_shapes)
> ```
> 
> Can anyone replicate this problem?
> 
> Thanks for your help.
Hi, we met the same problem as you yesterday, and later we reinstalled oonx, pip install onnx==1.5.0 to solve the segmentation fault. And in main.py, line 37, change tvm.relay.build to tvm.relay.build_module.build. Online_demo code works now.

**Comment by tonylins** [comment]
Hi @rachillesf, can you solve the problem using the solution from @karenz17 ?

**Comment by PuddingPengChen** [comment]
I solve the problem using the solution from @karenz17  in my laptop with ubuntu18.10. By the way, it seems not need opencv4 to run the online-demo, i just use the opencv3.5.

**Comment by zhangyuan1994511** [comment]
@karenz17 Hi, can you provided the version of tvm, onnx, LLVM, torch and python? Thank you very much!

**Comment by oscarkfpang** [comment]
Hi I experienced the same problem of Segmentation Fault while running the online demo. I tried the above solution of re-installing the onnx==1.5.0, but I haven't changed the line 37 of main.py. And the demo works now.

Here's my environment:
Ubuntu 18.04, Cuda 9.1
pytorch=1.2.0, torchvision=0.4.0 installed using pip3 install torch==1.2.0+cu92 torchvision==0.4.0+cu92 -f https://download.pytorch.org/whl/torch_stable.html
onnx=1.5.0
